Mathlib/Analysis/SpecialFunctions/Pow/Real.lean

```diff
@@ -513,6 +513,31 @@ lemma rpow_mul_intCast (hx : 0 ≤ x) (y : ℝ) (n : ℤ) : x ^ (y * n) = (x ^ y
 /-! Note: lemmas about `(∏ i in s, f i ^ r)` such as `Real.finset_prod_rpow` are proved
 in `Mathlib/Analysis/SpecialFunctions/Pow/NNReal.lean` instead. -/
 
+/-- `rpow` version of `Finset.prod_pow_eq_pow_sum`. -/
+theorem finset_prod_rpow_eq_rpow_sum
+    {ι} (s : Finset ι) (f : ι → ℝ) (hs : ∀ i ∈ s, 0 ≤ f i) (r : ℝ) (hr : 0 ≤ r) :
```

Comment:
Two small suggestions:
- I think writing `{ι : Type*}` is preferred (that's certainly what the remaining file does)
- you can make some of your variables implicit, e.g. f or r. (They are mentioned by `hs` and `hr`.)
Altogether, I arrive at
```suggestion
    {ι : Type*} (s : Finset ι) {f : ι → ℝ} (hs : ∀ i ∈ s, 0 ≤ f i) {r : ℝ} (hr : 0 ≤ r) :